nemo/collections/tts/modules/transformer.py

```diff
@@ -68,17 +70,17 @@ def __init__(self, d_model, d_inner, kernel_size, dropout, pre_lnorm=False):
             nn.Conv1d(d_inner, d_model, kernel_size[1], 1, (kernel_size[1] // 2)),
             nn.Dropout(dropout),
         )
-        self.layer_norm = nn.LayerNorm(d_model)
+        self.layer_norm = ConditionalLayerNorm(d_model, spk_emb_dim=d_model, condition=condition_lnorm)
```

Comment:
If condition_lnorm=False then it should use the original nn.LayerNorm definition. Otherwise checkpoints will break. And I don't want to use a custom written LN over the torch defined layer one unless it is absolutely required.

Reply:
I think when we set condition_lnorm=False. `ConditionalLayerNorm` with the same as `torch.nn.LayerNorm`. We have the affine weight and bias and call the F.layer_norm. https://github.com/pytorch/pytorch/blob/master/torch/nn/modules/normalization.py#L165-L195

Reply:
Fixed with inherit from `torch.nn.LayerNorm` to use original LN